google-cloud-firestore/src/test/java/com/google/cloud/firestore/TransactionTest.java

```diff
@@ -484,7 +484,7 @@ public String updateCallback(Transaction transaction) {
   @Test
   public void deleteDocument() throws Exception {
     doReturn(beginResponse())
-        .doReturn(commitResponse(0, 0))
+        .doReturn(commitResponse(2, 0))
```

Comment:
Forgive my ignorance, I'm not sure why this is changing.

Reply:
It's required because of the newly added Precondition: 
https://github.com/GoogleCloudPlatform/google-cloud-java/pull/2565/files#diff-837e4dce39e3a89ffb585a67cddc45cbR486

This test deletes two documents.

Reply:
got it, thank you